draft-ietf-openpgp-pqc.md

```diff
@@ -998,28 +1007,29 @@ scheme is as follows:
 
 ### Public-Key Encrypted Session Key Packets (Tag 1) {#ecc-mlkem-pkesk}
 
-The algorithm-specific fields consists of:
+The algorithm-specific fields consists of the output
+     of the encryption procedure described in {{ecc-mlkem-encryption}}:
 
  - A fixed-length octet string representing an ECC ephemeral public key in the
    format associated with the curve as specified in {{ecc-kem}}.
 
  - A fixed-length octet string of the ML-KEM ciphertext, whose length depends
    on the algorithm ID as specified in {{tab-mlkem-artifacts}}.
 
- - The one-octet algorithm identifier, if it is passed (in the case of a v3
-   PKESK packet).
+ - Only in the case of a v3 PKESK packet: a one-octet symmetric algorithm identifier.
 
- - A variable-length field containing the wrapped session key:
+ - The one-octet size of the following field;
 
-   - A one-octet size of the following field;
+ - The wrapped session key represented as an octet string.
 
-   - The wrapped session key represented as an octet string, i.e., the output
-     of the encryption procedure described in {{ecc-mlkem-encryption}}.
+Note that like in the case of the algorithms X25519 and X448 specified in
+{{I-D.ietf-openpgp-crypto-refresh}}, for the ML-KEM composite schemes, in the
+case of a v3 PKESK packet, the symmetric algorithm identifier is not encrypted.
+Instead, it is placed in plaintext after the `mlkemCipherText` and before the
+length octet preceding the wrapped session key.  In the case of v3 PKESK packets
+for ML-KEM composite schemes, the symmetric algorithm used MUST be AES-128,
+AES-192 or AES-256 (algorithm ID 7, 8 or 9).
```

Comment:
```suggestion
AES-192 or AES-256 (algorithm ID 7, 8 or 9).

An implementation MUST check if the key length matches the key algorithm identifier, and abort otherwise.
```

This would fully prevent cross-algorithm attacks

Reply:
Good suggestion, I adopted a slightly different variant of the sentence.